src/Makefile

```diff
@@ -950,6 +958,7 @@ $(TVOS_BUILD_DIR)/tvos/generated_sources: $(TVOS_BUILD_DIR)/tvos/generator.exe $
 		-d:TVOS                                                  \
 		--ns:ObjCRuntime                                         \
 		$(TVOS_APIS)                                            \
+		--target-framework=Xamarin.TvOS,v1.0                     \
```

Comment:
Good point, in theory case-insensitive comparison is done, but not everywhere. I'll fix it to be case-insensitive everywhere (and use the canonical case `Xamarin.TVOS` everywhere as well).